src/cmd/linuxkit/run_hyperkit.go

```diff
@@ -6,13 +6,75 @@ import (
 	"io/ioutil"
 	"net"
 	"os"
+	"os/exec"
 	"path/filepath"
 
 	log "github.com/Sirupsen/logrus"
 	"github.com/moby/hyperkit/go"
 	"github.com/satori/go.uuid"
 )
 
+var defaultVPNKits = []string{"vpnkit",
+	"/usr/local/bin/vpnkit",
```

Comment:
Hm no, I'll remove it - if you have a relatively new DfM installed it should be in path, and older versions wouldn't have the features we need.